### PR TITLE
Correct README

### PR DESCRIPTION
**Pull request by beckend**

### CODE REVIEW

**Comment by EnzoMartin** [comment]
Why are you forking the repo? You can just branch

**Comment by beckend** [comment]
I don't have write access.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codestyle",
-  "version": "20.0.0",
+  "version": "20.1.0",
```
Comment (EnzoMartin) [inline package.json]: Technically this is a `patch` version, but I'll allow it

Comment (beckend) [inline package.json]: Does npm allow patch bumps?

Comment (EnzoMartin) [inline package.json]: Yea

Comment (beckend) [inline package.json]: As a release that is

Comment (EnzoMartin) [inline package.json]: Yes

Comment (beckend) [inline package.json]: I never managed to do that, assholes

**Pull request merged**